Wikipedia/Code/UITextView+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+
+import Foundation
+
+extension UITextView {
+    func frame(of textRange: NSRange) -> CGRect? {
+        selectedRange = textRange
+        
+        var rect: CGRect?
+        if let uiTextRange = selectedTextRange {
+            rect = firstRect(for: uiTextRange)
+        }
+        
+        selectedRange = NSRange(location: 0, length: 0)
```

Comment:
Would it make sense to store the previously selected range in case this is used in a case where selection is available?

Reply:
@joewalsh agreed, done!